### ISSUE TITLE
[UX] System Updates - incorrect button label when an update has been selected

### ISSUE THREAD

**Issue by albanycomputers** [backdrop/backdrop-issues#5165]
**Description of the bug**

System Updates submit button does not change from "No updates selected" when an update has been selected.

![image](https://user-images.githubusercontent.com/1705388/130268016-52ad02f8-39dd-4eac-8fe5-21064b60b09f.png)

**Steps To Reproduce**

To reproduce the behavior:
1. Go to Configuration->System->System updates
2. Click on 'An update that is available'
3. Button does not change label

**Actual behavior**

Button does not change to reflect an update has been selected

**Expected behavior**

Button changes label when all or one update(s) are selected to reflect that an update will be installed.

**Additional information**

Add any other information that could help, such as:
- Backdrop CMS version: 1.19.2
- Web server and its version: ?
- Operating System and its version: Win 10
- Browser(s) and their versions: Chrome 92.0.4515.159

**Comment by olafgrabienski** [comment]
Thanks for the report! Interestingly I wasn't able to reproduce the issue (using Backdrop 1.19.3 and Firefox): When I select one or all available updates on my site, the button text changes to `Download selected updates` or `Download all updates` respectively.

<img width="564" alt="backdrop-select-one-update" src="https://user-images.githubusercontent.com/14188307/130279872-05529ac7-ff85-4509-8021-e8f6d6fe0424.png">

<img width="564" alt="backdrop-select-all-updates" src="https://user-images.githubusercontent.com/14188307/130279871-9aa15424-971c-439c-aa05-922b1cf1adf1.png">

**Comment by albanycomputers** [comment]
I've just tried on another site using Firefox and Chrome and the button label doesn't change.

**Chrome**
![image](https://user-images.githubusercontent.com/1705388/130293991-34c46307-4760-442e-a8ea-34374f920fdc.png)

**Firefox**
![image](https://user-images.githubusercontent.com/1705388/130294016-a258ae0d-3b16-4c49-86ed-fe496cb7a6bc.png)

**Comment by albanycomputers** [comment]
Just updated to 1.19.3 and still the same.... I've also cleared Caches multiple times.
![image](https://user-images.githubusercontent.com/1705388/130294338-e583ba7a-b3da-4b87-b013-2b426a7d7cf4.png)

**Comment by stpaultim** [comment]
@albanycomputers - I'm also not able to recreate this problem on a fresh site. I just did the following. 

1) I spun up a fresh install of Backdrop CMS using Lando (Using Chrome on Linux machine)
2) I downloaded an outdated version of the Webform module and installed it. 
3) I went to the updates page - admin/config/system/updates (initially, I see a gray button that says No Update Selected)

![updates-001](https://user-images.githubusercontent.com/3144571/130303609-ec82d0ef-3ab1-4c3d-91f7-ce96e53ef2e1.png)

4) I then check the box for Webform and the button immediately changes to blue with the text "Download Selected Updates."

![Updates-002](https://user-images.githubusercontent.com/3144571/130303680-a2c3c226-52a2-4996-9e66-9dcb1e6de395.png)

In your screenshots, it appears that the color has changed, but the text has not. Must be something specific to your situation.

**Comment by docwilmot** [comment]
Also works fine for me, WAMP on Windows, 1.20.x-dev.

**Comment by albanycomputers** [comment]
Very strange...

Just checked 5 sites and they did as I have reported.

Just checked another and it was on version 1.17 so I've updated it to 1.19.3 and this one now performs as one would expect.

Going to have a look at a few more of my sites.

**Comment by albanycomputers** [comment]
Just been experimenting with a site with only 11 modules installed...

I created a new site and started to install the modules one at a time.

**Installed the following and all worked fine...**
animate_css
backup_migrate
colorbox
eu_cookie_compliance
libraries
module_filter - v2.2.1 - old version so I could test
pretty_calendar - v1.0.0 - old version so I could test


Installed html5_upload and the button stopped working...  uninstalled and removed html5_upload and the button is still not working.

Started to work backwards and uninstalled - eu_cookie_compliance - and the button worked correctly

Installed 
html5_upload
token_filter
views_slideshow
views_slideshow_cycle2
webform

And the button worked correctly.

Installed eu_cookie_compliance - and it stopped working ...

Looks like it's eu_cookie_compliance that is causing the problem.

So even though "I agree" it must still be stopping something from working.  

I have ticked "/admin/config/system/eu-cookie-compliance" advanced - Exclude admin pages. - and the button starts working again.

Not sure what to try or check next...  so any suggestions would be appreciated.
